### PR TITLE
BBWP 126: use landscape category

### PR DESCRIPTION
**Pull request by yuki-nmi**
Measure table was updated with new measure scores for water buffering (effect_wb), and landscape-type-specific weighing factor is introduced.
Test functions are updated accordingly.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: BBWPC
 Type: Package
 Title: Calculator for BedrijfsBodemWaterPlan (BBWP)
-Version: 2.2.2
+Version: 2.2.3
```
Comment (KeesDoolNMI) [inline DESCRIPTION:4]: This should be version 2.3.0

**Review by KeesDoolNMI — CHANGES_REQUESTED** [review]
Nice work, a few final questions

File: DESCRIPTION (hunk 1 above)
Comment (yuki-nmi) [reply DESCRIPTION:4]: Indeed! I changed to 2.3.0, also in NEWS.md

File: dev/bbwp_measures2.csv
[the remark this answers is not in the document]
Comment (yuki-nmi) [reply dev/bbwp_measures2.csv:1]: now the script to make bbwp_measures table is moved to the Github Repo NMI-DATA_scripts. The products of the script is copied to the folder 'dev'

**Review by yuki-nmi — COMMENTED** [review]
Requested changes were made.

**Comment by yuki-nmi** [comment]
The syntax of melt function was fixed, so that it is now compatible for data.table 1.16.0.
Further, I found an error in the bbwp_measures.Rd (i.e. a unncessary row-name column was included), so I removed it and updated the test function.

**Comment by KeesDoolNMI** [comment]
Fantastic! Thanks for the effort 😄

**Pull request merged**